packages/api/tests/args.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { describe, it, expect } from "vitest";
+
+describe("arguments", function () {
```

Comment:
I don't follow what this test file is for

Reply:
It was me sanity checking `arguments` and TypeScript strict checking, will remove.